actors/witness.go

```diff
@@ -113,35 +113,33 @@ func (w *Witness) handlePublish(generic define.Generic) ([]byte, []byte, error)
 	case "roll call":
 		switch data["action"] {
 		case "create":
-			//return w.handleCreateRollCall(message, params.Channel, generic)
+			return w.handleCreateRollCall(message, params.Channel, generic)
 		case "state":
-
+			return nil, nil, define.ErrInvalidAction
 		default:
 			return nil, nil, define.ErrInvalidAction
 		}
 	case "meeting":
 		switch data["action"] {
 		case "create":
-			//return w.handleCreateMeeting(message, params.Channel, generic)
+			return nil, nil, define.ErrInvalidAction
```

Comment:
well we don't have any error in the protocol for not yet implemented. Hence we say the Action is invalid, as the message totally valid is.

But true, we can implement another "intern" error type notImplemented and return this one instead (would make the code more readable). Do you prefer we do it in this PR or in the next one? (as it will change many parts of the code)

Reply:
I would suggest adding a `ErrNotImplemented` in the working spec to facilitate cross-team collaboration. That won't lead to much code change for anyone.

As for how you want to handle that (this PR or a next one), that's up to you :)